### PR TITLE
[ID] Update "Chat Console"

### PR DESCRIPTION
**Pull request by wowcake**
(slightly) Updated the "Chat console " indonesian article.

- Some terms were translated as literal, which made it weird to understand, has now been reverted.
- Added some missing details.
- Slightly better wording.

### CODE REVIEW

File: wiki/Chat_Console/id.md [hunk 1]
```diff
@@ -1,40 +1,37 @@
----
-outdated: true
----
 
-# Konsol Obrolan Standar
+# Konsol Chat
 
 Di dalam osu! kamu dapat menekan tombol `Show Chat` di pojok kanan bawah layar atau dengan menekan tombol `F8` untuk membuka konsol obrolan.
```
Comment (mousesalt) [inline wiki/Chat_Console/id.md:4]:
```suggestion
Di dalam osu! kamu dapat menekan tombol `Show Chat` di pojok kanan bawah layar atau dengan menekan tombol `F8` pada keyboard untuk membuka konsol chat.
```

File: wiki/Chat_Console/id.md [hunk 2]
```diff
@@ -1,40 +1,37 @@
...
 
 ![Chat Console](img/Chatconsole1.png "Chat Console")
```
Comment (mousesalt) [inline wiki/Chat_Console/id.md:6]:
```suggestion
![Tampilan Konsol Chat](img/Chatconsole1.png "Tampilan Konsol Chat")
```

File: wiki/Chat_Console/id.md [hunk 3]
```diff
@@ -1,40 +1,37 @@
...
 
-- Tab ini hanya ada beberapa channel. Cukup klik Tab yang lainnya untuk pindah ke channel tersebut atau dengan mengklik tanda tambah `+` untuk membuka tab channel yang baru.
+- Tab ini hanya ada beberapa channel. Cukup klik Tab yang lainnya untuk pindah ke channel tersebut atau dengan mengklik tanda tambah bewarna kuning `+` untuk membuka tab channel yang baru.
```
Comment (mousesalt) [inline wiki/Chat_Console/id.md:8]:
```suggestion
- Daftar tab pada gambar tersedia beberapa kanal. Cukup klik tab lainnya untuk pindah ke kanal yang ingin dituju atau dengan mengklik tanda tambah berwarna kuning `+` untuk membuka tab kanal yang baru.
```

File: wiki/Chat_Console/id.md [hunk 4]
```diff
@@ -1,40 +1,37 @@
...
 - Warna dari masing-masing pengguna memiliki arti yang berbeda.
 
-| Warna |  |
+| Warna | Siapa?  |
 | :-- | :-- |
 | Putih | Kamu |
 | Kuning Pucat | Orang Lain |
```
Comment (mousesalt) [inline wiki/Chat_Console/id.md:14]:
```suggestion
| Kuning Pucat | Bukan pengguna osu!supporter |
```

File: wiki/Chat_Console/id.md [hunk 5]
```diff
@@ -1,40 +1,37 @@
...
 | Kuning | [osu!supporter](/wiki/osu!supporter) |
```
Comment (mousesalt) [inline wiki/Chat_Console/id.md:15]:
```suggestion
| Kuning | Pengguna [osu!supporter](/wiki/osu!supporter) |
```

File: wiki/Chat_Console/id.md [hunk 6]
```diff
@@ -1,40 +1,37 @@
...
-| Merah | Anggota [Tim Moderasi Global](/wiki/People/The_Team/Global_Moderation_Team) atau [Tim Penjamin Kualitas](/wiki/Glossary#quality-assurance-team) |
+| Merah | Anggota [Tim Moderasi Global (GMT)](/wiki/People/The_Team/Global_Moderation_Team) atau [Tim Penjamin Kualitas (NAT)](/wiki/Glossary#quality-assurance-team) |
 | Hijau | Baris yang berisi nama pengguna anda atau berbagai kata kunci yang anda tambahkan ke daftar highlight. Salinan dari pesan itu juga akan muncul di channel yang terdedikasi untuk itu yaitu, `#highlight`. |
```
Comment (mousesalt) [inline wiki/Chat_Console/id.md:17]:
```suggestion
| Hijau | Baris yang berisi nama pengguna kamu atau berbagai kata kunci yang ditambahkan ke daftar highlight. Salinan dari pesan itu juga akan muncul di kanal yang diperuntukkan untuk itu yaitu, `#highlight`. |
```

File: wiki/Chat_Console/id.md [hunk 7]
```diff
@@ -1,40 +1,37 @@
...
-## Konsol Obrolan yang Diperluas
+## Konsol Chat yang Diperluas
 
-*[osu!academy](/wiki/osu!academy) membuat dalam bentuk video [Episode 6 (6:52)](http://www.youtube.com/watch?v=cyYRl-a5xII) bersamaan dengan [Bermain Bersama](/wiki/Multi).*
+*[osu!academy](/wiki/osu!academy) membuat dalam bentuk video [Episode 6 (6:52)](http://www.youtube.com/watch?v=cyYRl-a5xII) bersamaan dengan [Multiplayer](/wiki/Multi).*
 
-Kamu dapat menekan `F9` atau klik tombol `Online Users` di bagian bawah kanan layar untuk membuka Konsol Obrolan yang Diperluas. Selain konsol obrolan, juga menampilkan informasi berbagai macam pengguna yang sedang daring osu!
+Kamu dapat menekan `F9` atau klik tombol `Online Users` di bagian bawah kanan layar untuk membuka Konsol Chat yang Diperluas. Selain konsol obrolan, juga menampilkan informasi berbagai macam pengguna yang sedang daring osu!
 
 ![Extended chat console](img/Chat_Console-Extended.png "Extended chat console")
```
Comment (mousesalt) [inline wiki/Chat_Console/id.md:32]:
```suggestion
![Chat konsol yang diperluas](img/Chat_Console-Extended.png "Chat konsol yang diperluas")
```

File: wiki/Chat_Console/id.md [hunk 8]
```diff
@@ -1,40 +1,37 @@
...
 
-Setiap pengguna yang masuk ke dalam osu! mempunyai panel pengguna yang ditampilkan konsol obrolan yang diperluas. Secara baku, panel pengguna akan memberikan informasi umum (nama, total skor ranked, peringkat, akurasi, jumlah bermain, dan avatar pengguna). Jika mouse diarahkan ke panel pengguna, panel pengguna akan memperlihatkan informasi yang berbeda (nama, peringkat, avatar jika ada, waktu lokal mereka, lokasi negara dan kota, dan kegiatan yang pengguna sedang lakukan).
+Setiap pengguna yang masuk ke dalam osu! mempunyai panel pengguna yang ditampilkan konsol chat yang diperluas. Secara default, informasi umum pemain akan ditampilkan (nama, total skor ranked, peringkat, akurasi, jumlah bermain, dan avatar pengguna) pada panel pengguna. Jika mouse diarahkan ke panel pengguna, panel pengguna akan menampilkan informasi yang berbeda (nama, peringkat, avatar jika ada, waktu lokal mereka, lokasi negara dan kota, dan kegiatan yang sedang dilakukan oleh pengguna).
```
Comment (mousesalt) [inline wiki/Chat_Console/id.md:34]:
```suggestion
Setiap pengguna yang masuk ke dalam osu! mempunyai panel pengguna yang ditampilkan pada konsol chat yang diperluas. Secara default, informasi umum pemain akan ditampilkan (nama, total skor ranked, peringkat, akurasi, jumlah bermain, dan avatar pengguna) pada panel pengguna. Jika mouse diarahkan ke panel pengguna, panel pengguna akan menampilkan informasi yang berbeda (nama, peringkat, avatar jika ada, waktu lokal mereka, lokasi negara dan kota, dan kegiatan yang sedang dilakukan oleh pengguna).
```

File: wiki/Chat_Console/id.md [hunk 9]
```diff
@@ -84,10 +81,10 @@ Tekan tombol angka atau menekan salah satu bar untuk:-
...
-| `/ignore [pengguna][@chp]` | Mengabaikan segala pesan dari pengguna yang telah dispesifikasikan pada sesi ini. Dengan menambahkan sebuah @ diikuti oleh huruf, c,h, dan/atau p, kamu mengabaikannya di obrolan, highlight, atau pesan pribadi secara berurut. | /ignore Amigo@chp | BanchoBot: You will no longer hear Amigo {chat} {highlights} {PM} (Konsol obrolanmu disetel untuk: mengabaikan teks apapun yang Amigo tulis \[c\], teks yang mungkin menghighlightmu dari Amigo \[h\] Pesan pribadi yang dikirim Amigo kepadamu \[p\]) |
+| `/ignore [pengguna][@chp]` | Mengabaikan segala pesan dari pengguna yang telah dispesifikasikan pada sesi ini. Dengan menambahkan sebuah @ diikuti oleh huruf, c,h, dan/atau p, kamu mengabaikannya di obrolan, highlight, atau pesan pribadi secara berurut. | /ignore Amigo@chp | BanchoBot: You will no longer hear Amigo {chat} {highlights} {PM} (Konsol chat kamu disetel untuk: mengabaikan teks apapun yang Amigo tulis \[c\], teks yang mungkin menghighlightmu dari Amigo \[h\] Pesan pribadi yang dikirim Amigo kepadamu \[p\]) |
 | `/j [channel]` or `/join [channel]` | Memasuki channel yang dispesifikasikan | `/join #lobby` | (Tab #lobby akan terbuka) |
```
Comment (mousesalt) [inline wiki/Chat_Console/id.md:85]:
```suggestion
| `/j [channel]` or `/join [channel]` | Bergabung ke spesifik kanal | `/join #lobby` | (Tab #lobby akan terbuka) |
```

File: wiki/Chat_Console/id.md [hunk 10]
```diff
@@ -84,10 +81,10 @@ Tekan tombol angka atau menekan salah satu bar untuk:-
...
 | `/p` or `/part` | Meninggalkan channel yang sedang kamu masuki (Perintah ini akan menutup jendela channel di tempat kamu mengetik perintah ini). | `/part` | n/a |
```
Comment (mousesalt) [inline wiki/Chat_Console/id.md:86]:
```suggestion
| `/p` or `/part` | Meninggalkan kanal yang sedang kamu masuki atau meninggalkan kanal. | `/part` | n/a |
```

**Review by mousesalt — COMMENTED** [review]
current phases are really odd to understand.
hopefully this is fine

**Comment by wowcake** [comment]
definitely agree on that part, was thinking to fix some of the things further ☹️

thanks for that ^^

**Review by mousesalt — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**